simple_locations/views.py

```diff
@@ -1,31 +1,29 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
 from django.conf import settings
-from django.core import serializers
 from django.db import IntegrityError
 from django.http import HttpResponse, HttpResponseRedirect
-from django.shortcuts import render, get_object_or_404
+from django.shortcuts import get_object_or_404, render_to_response
```

Comment:
Why render_to_response instead of render? The former was sort-of replaced by the latter in Django 1.3...